src/extension/features/accounts/hide-memos/components/hide-memos-button.jsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import { l10n, getToolkitStorageKey } from 'toolkit/extension/utils/toolkit';
+
+export const HideMemosButton = ({ toggleHiddenState }) => {
+  const notHidden = getToolkitStorageKey('hide-memos', true);
+  const label = notHidden ? l10n('app.show', 'is-checked') : l10n('app.hide', 'not-checked');
+
+  const toggleHidden = () => {
+    toggleHiddenState(!notHidden);
+    controllerLookup('application').send('closeModal');
+    document.getElementById('ember1434').remove();
+  };
+
+  return (
+    <div
+      title="Hide Memo Column"
+      id="ember1434"
```

Comment:
should be able to remove this.

Reply:
The title could maybe be removed, but the id is used to check if the button exists and if so destroy it before making another one with the a check or not.

Reply:
nm, that must have been a different reason my div was being stored in the model.  removing these attributes and the destroy method.